### PR TITLE
215: Verify webrev links before considering publication successful

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review this change which ensures that webrev links can be successfully reached before returning from the publication step. 

Best regards,
Robin

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

## Issue
[SKARA-215](https://bugs.openjdk.java.net/browse/SKARA-215): Verify webrev links before considering publication successful

## Approvers
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

File: bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/WebrevStorage.java [hunk 1]
```diff
@@ -141,6 +145,33 @@ private static void clearDirectory(Path directory) {
...
+                               .build();
+        while (Instant.now().isBefore(end)) {
+            var uncachedUri = uriBuilder.setQuery(Map.of("nocache", UUID.randomUUID().toString())).build();
+            var request = HttpRequest.newBuilder(uncachedUri)
+                                     .timeout(Duration.ofSeconds(30))
+                                     .GET()
+                                     .build();
+            try {
+                var response = client.send(request, HttpResponse.BodyHandlers.ofString());
+                if (response.statusCode() < 400) {
```
Comment (edvbld) [inline bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/WebrevStorage.java]:
```suggestion
                if (respons.statusCode() >= 200 && response.statusCode() < 400) {
```

Comment (rwestberg) [reply bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/WebrevStorage.java]: I don't think < 200's propagate outside of the http client implementation, so should be reasonably safe.

**Comment by rwestberg** [comment]
Thanks for reviewing!

**Pull request closed without merging**